basyx.aasrepository-http/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/http/TestAasRepositoryHTTP.java

```diff
@@ -187,11 +187,36 @@ public void removeNonExistingSubmodelReference() throws FileNotFoundException, I
 		assertEquals(404, deleteResponse.getCode());
 	}
 
+	@Test
```

Comment:
Update of nonExisting is missing as test case

Reply:
Added testcase for this